src/test/java/com/moodybluez/enterprise/EnterpriseApplicationTests.java

```diff
@@ -108,16 +109,17 @@ private void thenReturnMoodEntry() throws Exception {
 	@Test
 	void fetchEntryByDate_DateHasEntry() {
 		givenMoodDataAreAvailable();
+		whenEntryIsCompleted();
```

Comment:
This Test was incomplete according to me, it never entered an entry, either through "given" method or any other. So I used already defined method that does that to prevent Code duplication or DRY. 
Citation - "DRY stands for “Don's Repeat Yourself”." - [Baeldung](https://www.baeldung.com/java-clean-code)